GNUmakefile

```diff
@@ -25,6 +25,9 @@ all: $(programs)
 $(programs):
 	cargo build --target-dir="${TARGETDIR}" ${CARGO_ARGS}
 
+clean::
```

Comment:
Double quote means that multiple clean targets can be attached. It's good `Makefile` practice, esp. for `clean` because there might be multiple aspects to clean up.

+1 I will add the `.PHONY` notation.